library/Fission/Platform/Heroku/Provision.hs

```diff
@@ -39,11 +38,27 @@ data Request = Request
   , _uuid        :: UUID          -- ^ The unique identifier Heroku uses for the installed add-on. It corresponds with the id field in the Heroku Platform API.
   } deriving ( Eq
              , Show
-             , Generic
              )
 
 makeLenses ''Request
-$(deriveJSON lens_snake_case ''Request)
+
+instance ToJSON Request where
+  toJSON Request {..} = object
+    [ "callback_url" .= _callbackUrl
+    , "name"   .= _name
+    , "plan"   .= _plan
+    , "region" .= _region
+    , "uuid"   .= _uuid
+    ]
+
+instance FromJSON Request where
+  parseJSON = withObject "Heroku.Request" \obj -> do
+    _callbackUrl <- obj .: "callback_url"
+    _name        <- obj .: "name"
+    _plan        <- obj .: "plan"
+    _region      <- obj .: "region"
+    _uuid        <- obj .: "uuid"
+    return Request {..}
```

Comment:
Yeah, honestly these read well enough, despite the added boilerplate versus deriving them ourselves. Plus we get more flexibility, and the code is clearer than some abstract serializer on a generator, IMO.

Reply:
Agreed. Much prefer being explicit here